concordia/static/js/contribute.js

```diff
@@ -415,3 +415,44 @@ $tagEditor
 
         displayMessage('error', message, 'tags-save-result');
     });
+
+var hideTooltipCallback = function() {
+    // wait a couple seconds and then hide the tooltip.
+    var hideTooltip = function(tooltipButton) {
+        return function() {
+            tooltipButton.tooltip('hide');
+        };
+    };
+    setTimeout(hideTooltip($(this)), 3000);
+};
+var $copyUrlButton = $('#copy-url-button');
+$copyUrlButton.on('click', function() {
+    var $currentAssetUrl = $('#currentAssetUrl');
+    $currentAssetUrl.removeClass('d-none');
+    var currentAssetUrl = document.getElementById('currentAssetUrl');
+    currentAssetUrl.select();
+    var tooltipMessage = '';
+    try {
+        document.execCommand('copy');
+        // Show the tooltip with a success message
+        tooltipMessage = 'This link has been copied to your clipboard';
+        $currentAssetUrl.addClass('d-none');
+        $(this)
+            .tooltip('dispose')
+            .tooltip({title: tooltipMessage})
+            .tooltip('show');
+        $(this).on('shown.bs.tooltip', hideTooltipCallback);
```

Comment:
```suggestion
        $copyUrlButton
```

Or just continue the chain for this line, too.